labs/passport-adapter/LICENSE

```diff
@@ -0,0 +1,25 @@
+Copyright (c) IBM Corp. 2017,2019. All Rights Reserved.
```

Comment:
Would it be better to name the package as `authentication-passport`?

Reply:
+1 to use `authentication-passport`